kit/brain_board/index.md

```diff
@@ -13,21 +13,53 @@ The below will automatically calculate the latest version number, to be used whe
 Brain Board
 ===========
 
-<img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" />
-The Odroid U3+ Brain Board provided with your kit is what runs the code you write
-and controls the other peripheral boards.
+<img src="{{ site.baseurl }}/images/content/kit/brain.png" alt="A photo of a brain board" title="A brain board" class="right" style="max-width: 50%"/>
+The Brain Board provided with your kit is what runs the code you write and controls the other peripheral boards. It consists of a [Raspberry Pi 4B](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/) and a Student Robotics KCH HAT.
 
-### Flashing SD card
+The LEDs on the HAT display the current status of the robot and can be used to help debug your robot. There are also 3 RGB (Red, Green and Blue) LEDs that you can control from your code.
 
-In order to get access to the SD card, you will need to open up the case for the brain board. Pull the latch at the end of the case in order to hinge it open.
-The SD card is located on the underside of the board. Carefully open the Brain Board's plastic case, grab the SD card with your fingers and simply pull it out of the slot.
+## Board Diagram
+
+![brain board assembly diagram]({{ site.baseurl }}/images/content/kit/brain-diagram.png "The Brain Board Assembly")
+
+### Indicators
+
+| LED              | Meaning                 
+|------------------|-------------------------
+| 5V Power         | The board is powered    
+| 12V Power        | The board is powered    
+| Reverse Polarity | The 12V power is reversed
+| Boot Progress    | Progress Bar for Brain Boot Progress 
+| Code             | A USB containing code is plugged in
+| Comp             | The Robot is in Competition Mode
+| WiFi             | The Robot WiFi hotspot is running
+| ♥ (Heartbeat)    | Blinks when the Brain is running
+| Start            | The Robot is waiting to start
+| OK               | Shows the code status, see below table
```

Comment:
Maybe include the state these would be in after boot, as we do for other boards?

Reply:
I think we'd need to define the cutoff a bit more carefully. For many of these LEDs, it depends what's plugged into the Robot. For example, if a code USB was plugged in, the Code LED would be turned on before the Boot Progress bar finishes.

Reply:
I wonder if there's something we can do here though -- for example knowing whether or not the WiFi just hasn't come up _yet_ and you should wait longer for it vs something meaning that it's not going to come up is potentially useful to know.

Reply:
Theoretically, the final boot progress LED only turns on when all of the Astoria daemons are alive and working. I *think* in the case of `astwifid`, it only reports as running once it's determined whether to turn on the hotspot. (i.e, the WiFi LED light is on anyway)

Side note: There are tentative plans to get astwifid to connect to the hostapd control socket to check that a) hostapd is actually running; and b) allow us to only set the LED to solid once a client device has connected